### PR TITLE
Fixup language on help message to denote phases

### PR DESCRIPTION
**Pull request by timothysc**
**What type of PR is this?**
 /kind cleanup

**What this PR does / why we need it**:
Fixup language on help message to denote phases.  The help message to the user, was more confusing than anything. 

**Special notes for your reviewer**:
None

**Does this PR introduce a user-facing change?**:

```
NONE
```

/cc @kubernetes/sig-cluster-lifecycle-pr-reviews

### CODE REVIEW

**Comment by timothysc** [comment]
/hold  

I'll use this PR to bundle up other CLI issues.

File: cmd/kubeadm/app/cmd/phases/bootstraptoken.go [hunk 1]
```diff
@@ -61,7 +61,7 @@ func NewBootstrapTokenPhase() workflow.Phase {
 	return workflow.Phase{
 		Name:    "bootstrap-token",
 		Aliases: []string{"bootstraptoken"},
-		Short:   bootstrapTokenLongDesc,
+		Short:   "Generates bootstrap tokens used to join a node to a cluster",
```
Comment (neolit123) [inline cmd/kubeadm/app/cmd/phases/bootstraptoken.go]: i think this makes the var unused.

Comment (timothysc) [reply cmd/kubeadm/app/cmd/phases/bootstraptoken.go]: The var is used in the long description still.

Comment (timothysc) [reply cmd/kubeadm/app/cmd/phases/bootstraptoken.go]: fixing.

**Pull request merged**